packages/scripts/scripts/utils.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable global-require */
+
+const fs = require('fs');
+const get = require('lodash.get');
+const path = require('path');
+const which = require('which');
+
+/**
+ * Resolve the bin module executable path.
+ * This is from kcd-scripts (https://github.com/kentcdodds/kcd-scripts/blob/master/src/utils.js#L21)
+ * @param modName The bin module name
+ * @param executable The executable name (in case the executable has a different name)
+ * @param cwd The execution path
+ * @returns {*} The executable path
+ */
+function resolveBin(modName, { executable = modName, cwd = process.cwd() } = {}) {
+	let pathFromWhich;
+	try {
+		pathFromWhich = fs.realpathSync(which.sync(executable));
+	} catch (_error) {
+		// ignore _error
```

Comment:
Remove that try/catch if you don't intend of using the error.

Reply:
No it's to not throw the error from `which` module call.